### ISSUE TITLE
Mobile A11y Testing 11/1

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-badge#22]
Some of the badging element by ID badges have the small little icons in the upper right corner of the larger icon. Some of these are missing spoken labels, for instance the little one above the go forward button. So, when using Talkback to navigate, this information is just missing. Can we make sure to add labels here? 
"
